PhysicsTools/JetMCAlgos/interface/BasePartonSelector.h

```diff
@@ -14,7 +14,7 @@ class BasePartonSelector
 {
   public:
     BasePartonSelector();
-    ~BasePartonSelector();
+    virtual ~BasePartonSelector() {}
```

Comment:
The fix for this problem is to do `virtual ~BasePartonSelector()`. The definition of the function already exists somewhere.